src/main.cpp

```diff
@@ -374,9 +386,11 @@ void getCommandSchemaJson(JsonVariant json)
 
   JsonObject flash = properties.createNestedObject("flash");
   flash["type"] = "boolean";
+  flash["description"] = "used to set the speed of flashing mode and allows multiple controllers to be synced via the network";
```

Comment:
I personally think the flash mode should be removed. This is something that should be handled by the controller (i.e. NodeRED). NodeRED can simply send `{"mode":"fade"}` and `{"mode":"off"}` messages to achieve the same thing.

And if you want to sync them up then again, it should be handled in the logic layer. I think this flash mode is adding to the confusion and is not overly intuitive how it works.

What do you think?